cfme/tests/v2v/test_v2v_cancel_migrations.py

```diff
@@ -163,3 +164,55 @@ def _get_plan_status_and_cancel():
 
         # Test3: Check if instance has any ports attached
         soft_assert(provider.mgmt.get_ports(uuid=server.uuid))
+
+
+@pytest.mark.tier(1)
+def test_retry_migration_plan(appliance, provider, mapping_data_vm_obj_mini, request):
+    """
+    Test to cancel migration and then retry migration
+    Polarion:
+        assignee: sshveta
+        initialEstimate: 1/4h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+        casecomponent: V2V
+    """
+    migration_plan_collection = appliance.collections.v2v_migration_plans
+    migration_plan = migration_plan_collection.create(
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()),
+        description="desc_{}".format(fauxfactory.gen_alphanumeric()),
+        infra_map=mapping_data_vm_obj_mini.infra_mapping_data.get("name"),
+        target_provider=provider,
+        vm_list=mapping_data_vm_obj_mini.vm_list
+    )
+
+    @request.addfinalizer
+    def _cleanup():
+        migration_plan.delete_completed_plan()
+    assert migration_plan.wait_for_state("Started")
+    request_details_list = migration_plan.get_plan_vm_list(wait_for_migration=False)
+    vms = request_details_list.read()
```

Comment:
Since the test is to cancel then retry the plan - shouldn't these steps be part of setup (i.e. in a fixture)?